near-token-factory/src/lib.rs

```diff
@@ -26,11 +28,21 @@ enum StorageKey {
     TokenMap,
 }
 
+#[derive(AccessControlRole, Deserialize, Serialize, Copy, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub enum AclRole {
+    UpdateTokenBinary,
+    UpdateTokenAdmin,
```

Comment:
```suggestion
    TokenUpdater,
```

For every Acl role there are admins and grantees, so speaking about admins for `AclRole::UpdateTokenAdmin` might be confusing? I'd try to avoid using `Admin` in the name of a role.

Reply:
Done in ccd84c46af2278ef9b81fb21dcfa824d28ea41c6